### ISSUE TITLE
[Neon staging] Liquescents and obliques look weird

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/Neon2#1236]
When I uploaded Einsie 138v to Neon staging, both obliques and liquescents separated from their stems:

This is the file: [CH-E_611_138v.mei.zip](https://github.com/user-attachments/files/16279925/CH-E_611_138v.mei.zip)

And this is what it looks like: 

<img width="909" alt="liquescents and obliques on staging" src="https://github.com/user-attachments/assets/c69943e7-fd9a-429c-a479-d9d14ea8ca98">

I looked at the mei, and I can't tell what's wrong with the obliques. With the liquescents, however, I noticed that a second `liquescent` line appears under the first one when I upload the file to staging.

So this is the original encoding of the liquescent neume:
```
<neume xml:id="nns7rm4">
    <nc xml:id="m-02f2cef8-d6dc-406d-81d4-2a61de1a0c05" facs="#m-7d690635-ae09-4304-896f-8b93f9a6864c" oct="4" pname="c" curve="a">
      <liquescent xml:id="m-903fa7ce-9861-46b5-b8d1-bd24380386c0"/>
    </nc>
</neume>
```

And this is after upload:
```
<neume xml:id="nns7rm4">
    <nc xml:id="m-02f2cef8-d6dc-406d-81d4-2a61de1a0c05" facs="#m-7d690635-ae09-4304-896f-8b93f9a6864c" oct="4" pname="c" curve="a">
      <liquescent xml:id="m-903fa7ce-9861-46b5-b8d1-bd24380386c0"/>
      <liquescent xml:id="m-b9d67f8b-d3f5-41a7-9817-3d77108aabf5"/>
   </nc>
</neume>
```

I checked previous issues I wrote about staging and obliques looked fine, so this appears to be a relatively recent phenomenon.

**Comment by yinanazhou** [comment]
The changes has been merged!

**Comment by JoyfulGen** [comment]
Liquescents are doing a funny thing again! Some of them are fine, but some of them are appearing as teeny tiny puncta. Behold:

<img width="970" alt="148r smol neumes" src="https://github.com/user-attachments/assets/60d34343-eb75-4b7d-82e0-2725e7ca7851">

If I use the neume component selector and try to change their head shape, nothing happens. However, I can delete them and input a new liquescent that looks totally normal.

I don't know why some liquescents are fine and some aren't! I tried looking at the MEI and I can't find the difference... But I can tell you Einsie 148r has a lot of the tiny liquescents:

[CH-E_611_148r.mei.zip](https://github.com/user-attachments/files/16509447/CH-E_611_148r.mei.zip)

**Comment by annamorphism** [comment]
aw, little baby puncta!! (sorry, I have no insight into the issue...)

**Comment by JoyfulGen** [comment]
Small addendum: the fact that using the "Change Head Shape" function doesn't work is actually due to this issue https://github.com/DDMAL/Neon/issues/1239.

**Comment by yinanazhou** [comment]
Hi! This is because in this file, the parent `<nc>` of the `<liquescent>` doesn't have an associated `@curve` value for some reason. Does this happen to a lot of the files?

**Comment by JoyfulGen** [comment]
@yinanazhou I'd say it was happening 1-4 times in every file I was reviewing (which are in the 140-160 range of Einsie)

**Comment by yinanazhou** [comment]
Hi @JoyfulGen, is this still happening in neon?

**Comment by JoyfulGen** [comment]
@yinanazhou yes and no! The exact behaviour is not happening, but liquescents are still being a tiny bit weird:

If I select any neume component and change its head shape to Liquescent C or Liquescent A, the neume component becomes a punctum. If I do any action, it automatically becomes the correct liquescent.

Interestingly, in the MEI the liquescent is correct as soon as the head shape is changed. It just doesn't show up in Neon for some reason! 

It looks like this only happens for liquescents, but it does happen systematically.

**Comment by yinanazhou** [comment]
Hi @JoyfulGen this should be fixed now :)

**Comment by JoyfulGen** [comment]
@yinanazhou indeed it is! I'll close the issue.
